### ISSUE TITLE
Crop should mirror orientation changes

### ISSUE THREAD

**Issue by emko** [darktable-org/darktable#17498]
**Is your feature request related to a problem? Please describe.**
I have set a shortcut to flip my image while editing so i can switch back and forth quickly, its something i have done with drawing,painting etc to see the image in a different view, the issue is if there is any cropping applied when doing any orientation changes the crop does not follow the orientation changes so you have to edit the crop.

**Describe the solution you'd like**
when changing any of the orientation settings the crop should be adjusted to match, kind of  like how every other software does it there shouldn't be a need to readjust your crop anytime you rotate/flip etc your image

thanks

**Comment by ralfbrown** [comment]
Duplicate of #11614.

TL;DR: move orientation above crop if you want to use it for checking different views.

**Comment by emko** [comment]
> Duplicate of #11614 and #14788.
> 
> TL;DR: move orientation above crop if you want to use it for checking different views.

sorry, i though i did ask i tried searching for already existing issue, should i remove this one?

also you can't move orientation module. I tried other modules to see and retouch and liquify understands when orientation changes it follows but vignetting module does not behaves same as crop
